tests/test_filters.py

```diff
@@ -4,7 +4,7 @@
 from email.utils import formatdate
 
 from nyaa.routes import (_jinja2_filter_rfc822, _jinja2_filter_rfc822_es, get_utc_timestamp,
-                         get_display_time, timesince)
+                         get_display_time, timesince, filter_truthy, category_name)
```

Comment:
And `utils.flatten_dict()` isn't used since I changed the file list layout. :wink:
That's a good point, maybe in another PR though? I only tested the function. I do not want to edit templates and such in this PR.